blocks/donation-form/class-give-donation-form-block.php

```diff
@@ -172,6 +178,14 @@ public function render_donation_form( $attributes ) {
 		$parameters['display_style']         = $attributes['displayStyle'];
 		$parameters['continue_button_title'] = trim( $attributes['continueButtonTitle'] );
 
+		$class_names = Html::classNames(
+			isset( $attributes['className'] ) ? $attributes['className'] : null
+		);
+
+		add_filter( 'give_form_wrap_classes', function( $classes ) use ( $class_names ) {
+			return Html::classNames( $classes, $class_names );
```

Comment:
@mhimon this function should return an array.
I tested this with [Multi-step form template](https://givewp.com/documentation/core/give-forms/form-templates/) and classes are not added to the form wrapper.

Reply:
Maybe problem with that filter hook

Reply:
@mhimon this is not an issue with the filter. `Html::classNames` returns a string of class names instead of an array. 

https://github.com/impress-org/givewp/blob/452aad332e35f65767a021314d9630bf6ce2fa5d/includes/class-give-donate-form.php#L876